set.mm

```diff
@@ -182999,6 +183082,13 @@ infinite descent (here implemented by strong induction).  This is
       copab cxp brel ) ABFFGGCHZFIDHZFIJEHUAKLUBMEFNZJCDRFFSCDEOUCCDFFPQT $.
   $}
 
+  $( If a positive integer divides another integer, then the remainder upon
+     division is zero.  (Contributed by AV, 3-Mar-2022.) $)
+  dvdsmod0 $p |- ( ( M e. NN /\ M || N ) -> ( N mod M ) = 0 ) $=
+    ( cz wcel wa cn cdvds wbr cmo co cc0 dvdszrcl adantl dvdsval3 biimpd expcom
+    wceq wi impd mpcom ) ACDZBCDZEZAFDZABGHZEZBAIJKQZUEUCUDABLMUBUFUGRUAUBUDUEU
+    GUDUBUEUGRUDUBEUEUGABNOPSMT $.
+
```

Comment:
This statement is stronger than `~dvdsval3`, I guess it may minimize theorems using `~dvdsval3`, did you check that?

Reply:
There are 25 proofs using ~dvdsval3, but none of them can be minimized (automatically). Maybe some of them can be shortend manually. I will pick some and have a look (-> my TODO- list)